Makefile

```diff
@@ -10,14 +10,14 @@ webui:
 
 containers:
 	(cd $(INTERNALSERVICES)/volume-stage-in && docker build -t volume-stage-in .)
-	(cd $(INTERNALSERVICES)/volume-filelist && GOOS=linux GOARCH=amd64 go build && docker build -t volume-filelist .)
+	(cd $(INTERNALSERVICES)/volume-filelist && GOOS=linux GOARCH=amd64 go build -ldflags -s && docker build -t volume-filelist .)
```

Comment:
Please define `GOFLAGS=-ldflags -s` at the top of the makefile and use `go build $(GOFLAGS)` where needed, this way we can change the flags once.